### DISCUSSION TITLE
Split DisableOnMove into DisableOnMove and DisableOnHeadTurn for AFK status

### DISCUSSION THREAD

**Discussion by NeuTraLZero** [Zrips/CMI#4220] [Feature Requests]
**Description of feature/improvement request:**
Split DisableOnMove into DisableOnMove and DisableOnHeadTurn for AFK status
---
### Detailed explanation of what it should do, when and in what situation: ###
Basically right now, a player can go off their AFK status by simply turning their head only a very slight angle, but if DisableOnMove is set the false, the player would be able to walk around without changing the AFK status.
### Basic explanation as to why this feature is needed and/or useful to have: ###
This change would basically split these two to prevent some exploits by simply turning their head but also allow them to go off AFK status by moving around

**Comment by CloudeLecaw** [comment]
this makes sense. player does not go afk when he is moving his head 
Your request is simliar to this old one where eppl have problem with AFK fishing
https://github.com/Zrips/CMI/issues/3259

There i asked for:
```
 # Disables afk on leftclick
  DisableOnLeftClick: true
 # Disables afk on rightclick
  DisableOnRightClick: true
 # Disables afk on LookArround
  DisableOnLookArround: true
 # Disables afk when Fishing
  DisableWhenFishing: true
```

what may was missing

```
  # Disables afk when sneeking
   DisableWhenSneak: true
```


Now lets wait for Zrips to tell us that this is currently not possible by the minecraft code.. ;)

**Comment by Zrips** [comment]
Well @CloudeLecaw is some what correct, this is not really possible, but not for the reasons he said. All is based on PreventPushing option, if this is set to true then we will disable afk when player moves his head, this is needed due to how PreventPushing works in general, which basically disables player movement. So as player cant no longer move (outside block area), we have to disable afk mode when player looks around or performs command. 
Unless i'm missing entire point here. Can @NeuTraLZero provide afk config section so we will be on same page?
